server/config/routes.js

```diff
@@ -37,9 +37,13 @@ var accounts        = require('../controllers/accounts');
 var auth            = require('../controllers/auth'),
     projects        = require('../controllers/projects'),
     users           = require('../controllers/users'),
-    analytics       = require('../controllers/analytics');
+    analytics       = require('../controllers/analytics'),
+    budget          = require('../controllers/budget');
 
-var patient = require('../controllers/patient');
+var patient         = require('../controllers/patient');
+
+// Middleware for a particular route
```

Comment:
You might want to say "middleware for handling uploads".  The current documentation in not very specific.